Dockerfile

```diff
@@ -2,9 +2,14 @@
 # Runs service on port 80.
 # Healthchecks service up every 5m.  
 
-FROM python:3.7
+FROM python:3.9
 RUN apt update ; apt install -y rsync
 RUN pip install pipenv uvicorn[standard]
+ENV SEQREPO_ROOT_DIR=/usr/local/share/seqrepo/2021-01-29
+ENV GENE_NORM_DB_URL=http://dynamodb:8001
+ENV AWS_ACCESS_KEY_ID = 'DUMMYIDEXAMPLE'
+ENV AWS_SECRET_ACCESS_KEY = 'DUMMYEXAMPLEKEY'
+ENV AWS_DEFAULT_REGION = 'us-west-2'
```

Comment:
Maybe we should consider having these as ARGs.

Reply:
These values are required during container runtime as well. It would be better to continue with ENV unless we are planning for build time customisation.